packages/wheels_driver/src/wheels_driver_node.py

```diff
@@ -50,7 +51,17 @@ def __init__(self, node_name):
             "~emergency_stop", BoolStamped, self.cbEStop, queue_size=1)
 
         self.log("Initialized.")
-
+        
+        #subscriber to Lightadjustment
+        self.motorscale = 1
+        self.sub_light = self.subscriber ("Light_Adjustment",Light_Adjustment, self.cbmotorscale,queue_size=1)
+
+        
+    def cbmotorscale(self, data):
+        self.motorscale = data.motorscale
```

Comment:
perhaps this can be a more general thing, where we reserve a topic for realtime gain. This does not have to be light-specific, can be general enough so that anybody can slow down and speed up the bot.